### PR TITLE
[merged] More run regressions

### PR DESCRIPTION
**Pull request by baude**

### CODE REVIEW

File: Atomic/objects/image.py [hunk 1]
```diff
@@ -157,7 +157,9 @@ def populate_remote_inspect_info(self):
         self.labels = remote_inspect_info.get("Labels", None)
         self.release = self.get_label('Release')
         self.version = self.get_label('Version')
-        self.id = no_shaw(remote_inspect_info['id'])
+        self.id = remote_inspect_info['id']
+        if remote_inspect_info['id']  is not None:
```
Comment (rhatdan) [inline Atomic/objects/image.py]: Why not 
if not self.id

Comment (baude) [inline Atomic/objects/image.py]: The syntax wouldn't be right.  I did change to:
```
if self.id is not None:
```

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Comment by miabbott** [comment]
Well, this has already been merged, but I ran a couple of tests with `atomic pull` and `atomic run` without issue.  Appears to fix #838 and #844

**Comment by rhatdan** [comment]
Please close them if you are happy.

**Pull request closed without merging**